### ISSUE TITLE
No module named 'detectron2.config'

### ISSUE THREAD

**Issue by Kana-alt** [google/lasr#8]
I built my environment with docker.
Therefore, I use the following command  to get segmentations.

docker run -v $(pwd):/lasr --gpus all lasr bash -c 'cd lasr/detectron2; source activate lasr; python mask.py pika . /detectron2; cd -'

Then I get the following error.

Traceback (most recent call last):
  File "mask.py", line 23, in <module>
    from detectron2.config import get_cfg
ModuleNotFoundError: No module named 'detectron2.config'

detectron2 is installed with a folder created in the parent directory of preprocess.
Is it because I am using docker that I am getting this error?

**Comment by gengshan-y** [comment]
Yes, the provided docker script would not install and clone detectron2. You may need to modify the docker file to do so.
I'll try to push a fix. In the meanwhile, you may use the conda environment to extract segmentations.
